src/MSLGenerator.h

```diff
@@ -88,10 +88,6 @@ class MSLGenerator
     Target          m_target;
     
     bool            m_error;
-    
-    // TODO: Temporary hack to access uniforms struct members
-    HLSLBuffer* m_per_pass_buffer;
```

Comment:
Sure - but there is no need to special-case these two names in the translator. My understanding is that you already use something like `cbuffer per_item { float4 field1; ... }` - with this change the only other change you'll need to keep bindings the same is specify a register, like `cbuffer per_item: register(b0) { float4 field1; }` (otherwise it's first come first served, as it is with textures in your version). If you parse the argument table on the Metal side then it'll just work without any changes I think - per_item/per_pass would be assigned to different slots (see ParseRegister)